test-specifications/cryptoService/accountCreateTransaction.md

```diff
@@ -0,0 +1,156 @@
+| 3       | Creates an account with a valid ED25519 private key                                           | key=<VALID_ED25519_PRIVATE_KEY>         | The account creation succeeds.                                                 | N                 |
+| 4       | Creates an account with a valid ECDSAsecp256k1 private key                                    | key=<VALID_ECDSA_SECP256K1_PRIVATE_KEY> | The account creation succeeds.                                                 | N                 |
+| 5       | Creates an account with a valid KeyList of ED25519 and ECDSAsecp256k1 private and public keys | key=<VALID_KEYLIST>                     | The account creation succeeds.                                                 | N                 |
+| 6       | Creates an account with a valid KeyList of nested Keylists (three levels)                     | key=<VALID_NESTED_KEYLIST>              | The account creation succeeds.                                                 | N                 |
+| 7       | Creates an account with no key                                                                |                                         | The account creation fails with a KEY_REQUIRED response code from the network. | N                 |
+| 8       | Creates an account with an invalid key                                                        | key=<INVALID_KEY>                       | The account creation fails with an SDK internal error.                         | N                 |
+
+### **Initial Balance:**
+
+- The initial number of tinybars to put into the account.
+
+| Test no | Name                                                                                | Input                                             | Expected response                                                                             | Implemented (Y/N) |
+|---------|-------------------------------------------------------------------------------------|---------------------------------------------------|-----------------------------------------------------------------------------------------------|-------------------|
+| 1       | Creates an account with an initial balance                                          | key=<VALID_KEY>, initialBalance=100               | The account creation succeeds and the account contains 100 tinybar.                           | N                 |
+| 2       | Creates an account with a negative initial balance                                  | key=<VALID_KEY>, initialBalance=-1                | The account creation fails with an INVALID_INITIAL_BALANCE response code from the network.    | N                 |
+| 3       | Creates an account with an initial balance higher than the operator account balance | key=<VALID_KEY>, initialBalance=1,000,000,000,000 | The account creation fails with an INSUFFICIENT_PAYER_BALANCE response code from the network. | N                 |
+
+### **Receiver Signature Required:**
```

Comment:
Isn't false the default value? Do we need a test case for that?

Reply:
I think its about the decision if we should make it verbose or no. I mean adding tests to follow:

> The inputs for each test are a range of valid, minimum, maximum, negative and invalid values for the method.

Reply:
Yeah I guess it's probably better to be more verbose than not even if it might be a bit overkill. I'll add testing the default values for each property.

Reply:
Added test cases for default cases.